designer/fitting/dwipi.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, imPath):
             assert('File in path not found. Please locate file and try '
                    'again')
         tqdm.write('Image ' + fName + '.nii loaded successfully')
+        if not isinstance(nthreads, int):
+            assert('Variable nthreads need to be an integer')
```

Comment:
That is not how asserts should be used. `assert` does not belong in production code, it belongs in tests.

Reply:
Thanks for bringing this up. Made the changes